### PR TITLE
Ground metric

### PR DESCRIPTION
**Pull request by maikia**
calculates the distances between parcels, originally written by @hichamjanati

### CODE REVIEW

**Comment by maikia** [comment]
@hichamjanati could you pls have a look. do you agree with the changes?

File: ground_metric.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+                                 subjects_dir=subjects_dir)
+    tris = src[0]["use_tris"]
+    vertno = src[0]["vertno"]
+    points = src[0]["rr"][vertno]
+    D = mesh_all_distances(points, tris)
+    n_vertices = len(vertno)
+
+    mne.datasets.fetch_aparc_sub_parcellation(subjects_dir=subjects_dir,
+                                              verbose=True)
+    labels = mne.read_labels_from_annot(subject, annot, 'both',
```
Comment (hichamjanati) [inline ground_metric.py]: Are we doing source localization on both hemispheres ? if so I should adapt the computation of the ground metric, so far it's restricted to `lh`

Comment (hichamjanati) [reply ground_metric.py]: Done

**Comment by hichamjanati** [comment]
The plot looks good, good to go for me

**Comment by agramfort** [comment]
can you share what you see?

>

**Comment by maikia** [comment]
I am getting some Mayavi error when running `plot_ground_metric.py`
but here is the fig generated by @hichamjanati 
ground_metric

**Comment by hichamjanati** [comment]
did not push the right one, here 
ground_metric

distances between a label in the superior temporal sulcus and the other labels

**Comment by agramfort** [comment]
nice !

>

**Comment by maikia** [comment]
Thanks @hichamjanati

**Pull request merged**